iina/PrefKeyBindingViewController.swift

```diff
@@ -325,11 +326,14 @@ class PrefKeyBindingViewController: NSViewController, PreferenceWindowEmbeddable
   func saveToConfFile(_ sender: Notification) {
     let predicate = mappingController.filterPredicate
     mappingController.filterPredicate = nil
-    let keyMapping = mappingController.arrangedObjects as! [KeyMapping]
+    let keyMappings = mappingController.arrangedObjects as! [KeyMapping]
+    for mapping in keyMappings {
+      mapping.rawKey = KeyCodeHelper.escapeReservedMpvKeys(mapping.rawKey)
+    }
```

Comment:
No need to change, but could use `forEach` instead of `for`:
```swift
keyMappings.forEach { $0.rawKey = KeyCodeHelper.escapeReservedMpvKeys($0.rawKey) }

```

Reply:
Almost worked for me.

With these changes the `.conf` file now contains:
```text
SHARP quit
SPACE quit
```
The key bindings all look correct in settings. Looks as expected when I toggle `Display raw values`. When I press space IINA quits. **BUT** when I press # IINA responds with the unrecognized key sound and does not quit.

Reply:
> No need to change, but could use `forEach` instead of `for`:
> 
> ```swift
> keyMappings.forEach { $0.rawKey = KeyCodeHelper.escapeReservedMpvKeys($0.rawKey) }
> ```

I actually prefer to use a traditional `for` loop instead of a `forEach`, unless there's a really good reason to use it. I like having named variables and being able to introduce `break`, and maybe there's some jadedness from using debuggers which can't handle closures as well and make it harder to debug. I get the sense that some developers tell themselves they're making their code more efficient by reducing the number of lines written, but I think usually they're just increasing the bug density.

> BUT when I press # IINA responds with the unrecognized key sound and does not quit.

Well...yikes. I dug into this and found a whole bunch of bugs.... will comment more below.

Reply:
Yes, that is why I mentioned no need to change. I definitely will use a traditional `for` if the situation makes it useful to be able to use `break` or `continue`.